cmd/entrypoint-wrapper/main.go

```diff
@@ -193,7 +193,7 @@ func execCmd(argv []string) error {
 		for {
 			select {
 			case <-ctx.Done():
-				break
+				return
```

Comment:
A `break` here just breaks the select and not the loop

Reply:
:facepalm: